### ISSUE TITLE
Ordering attribute columns in the views

### ISSUE THREAD

**Issue by driebe** [zammad/zammad#523]
Hi,

as far as I know, there is currently no possibility of changing the order of attribute-colums shown in the views. I think it would be good if admins (or maybe even users) have the possibility of either changing the order of the columns via the settings (or even better: by just drag-&-dropping the columns from left to right). 

E.g.: Activating the "type" attribute places it at the far column of the views, my personal preference would be to have it in front (or after the ticket headline). I guess everybody will have their own preference...

Regards
Daniel

**Comment by dschuette** [comment]
Agreed, we also miss such a sort-column feature. Ideally this would be drag'n'drop and available to all admins/agents (at least).

**Comment by HamsterNL** [comment]
Is changing the order of the attribute columns in the views still not possible?

Maybe it's time to assign someone to this task?

**Comment by MrGeneration** [comment]
> Is changing the order of the attribute columns in the views still not possible?

It is not.

![Image](https://github.com/user-attachments/assets/d20c1259-1e4a-4256-b3fe-9d9d0ebbf050)

**Comment by HamsterNL** [comment]
Ok, that's unfortunate.

This ticket is 8 years old and I was hoping I just missed something obvious in Zammad (like drag & drop columns to change the order).

I hope this functionality will be added in 2025 👍
